reana/reana_dev/git.py

```diff
@@ -199,8 +201,11 @@ def git_create_release_commit(
     if modified_files:
         run_command(f"git add {' '.join(modified_files)}", component)
 
+    commit_msg = (
+        f"chore({base}): release {next_version}{get_commit_pr_suffix(component)}"
```

Comment:
BTW one meta comment related to the release news.

Currently this PR would advertise:

```
fix(reana-dev): create commits that conform to conventional style
```

This does not expose the `reana-dev` sub commands that this change improves.

If we would like to include those (for analogy with #765), perhaps the commit should be split in two, advertising something like:

```
fix(reana-dev): `git-create-release-commit` use conventional commit style
fix(reana-dev): `git-upgrade-shared-modules` use conventional commit style
```

But we are probably hitting the line length limits here...

Reply:
From my point of view this was a "general" change to the whole reana-dev to conform to the conventional commit style we have agreed on. Not sure how to also include the name of the command in the commit title while also staying under the character limit, unfortunately.